lang/java/reef-examples/src/main/java/org/apache/reef/examples/hello/HelloDriver.java

```diff
@@ -68,6 +68,31 @@ public void onNext(final StartTime startTime) {
   }
 
   /**
+   * Handles the StartTime event: Fails intentionally.
+   */
+  public final class FailureToStartHandler implements EventHandler<StartTime> {
```

Comment:
This shouldn't be in the simple example. Maybe we can turn this into the nucleus of the integration test we'll need eventually?